packages/v4/src/content/design-guidelines/content/capitalization.md

```diff
@@ -5,29 +5,78 @@ hideTOC: true
 showTitle: true
 ---
 
-Keep your capitalization consistent, and follow the capitalization rules your company enforces for your products.
+Consistent capitalization adds clarity and consistency across your products' UIs.
 
-Sentence case capitalizes only the first word of each sentence and proper nouns. On PatternFly, we use sentence case capitalization for all text elements, except the following:
-- Product names (Red Hat Virtualization, PatternFly, Red Hat CloudForms)
-- Proper nouns for individuals, organizations, and locations.
-- Acronyms and initialisms (URL, HTML, CSS, JS)
+## Capitalization types
 
-**Note**: Hyperlinked articles in a UI are usually not displayed as titles. Instead, the copy typically reads something along the lines of, “Learn more about [tool name].”
+* **Title case**: Title case capitalizes the first letter of each word in a sentence except for smaller words like articles and conjunctions that have fewer than four letters. 
 
-## Capitalization for tools outside your product portfolio
-If you’re referencing tools outside your company’s product portfolio, write the product names as they appear in the respective company’s documentation. For example, if you’re referencing a product in AWS and Amazon capitalizes it, then capitalize it in your writing.
+    <br />Example: "PatternFly Design Resources With Tips and Best Practices"
```

Comment:
This shows up wrong in the preview.
Try <br> instead of  <br/> 
![image](https://user-images.githubusercontent.com/51165119/89815543-f97f8680-db12-11ea-8756-27d32004a4d1.png)